app/Http/Controllers/FrontendTransportController.php

```diff
@@ -42,6 +42,22 @@ public function TrainStationboard(Request $request) {
         );
     }
 
+    public function StationByCoordinates(Request $request)
```

Comment:
I recommend renaming it "getStationByCoordinates", even if other method names are in this format. See my general answer to your pull request for further information.